cmake/Summary.cmake

```diff
@@ -172,7 +172,7 @@ function(caffe2_print_configuration_summary)
   if(${USE_NCCL})
     message(STATUS "    USE_SYSTEM_NCCL     : ${USE_SYSTEM_NCCL}")
   endif()
-  message(STATUS "  NVSHMEM_LIB_DIR       : ${NVSHMEM_LIB_DIR}")
+  message(STATUS "  Found NVSHMEM         : ${NVSHMEM_INCLUDE_DIR}")
```

Comment:
This is printed even if   `message(STATUS "  Found NVSHMEM         : ${NVSHMEM_INCLUDE_DIR}")` is empty...

Reply:
Empty means not found. 
It works on my machine tho:
```
...
--   USE_NCCL              : ON
--     USE_SYSTEM_NCCL     : OFF
--   Found NVSHMEM         : /usr/local/include
--   USE_NNPACK            : 0
--   USE_NUMPY             : ON
...
```